smsapi/smsapi.csproj

```diff
@@ -10,14 +10,15 @@
   </PropertyGroup>
   <PropertyGroup>
     <PackageId>SMSAPI.pl</PackageId>
-    <PackageVersion>1.0.9</PackageVersion>
+    <PackageVersion>1.1.0</PackageVersion>

```

Comment:
Let's make it 2.0.0, there are too many changes made.

Reply:
Also there any changes of README needed, you've added new dependency etc.

Reply:
ded0363